tests/e2e/managed-reconcile/01-assert.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
```

Comment:
In this assertion I would check if the collector config map (check if the jaeger is there - it was not reverted by the operatr)

Reply:
Done